docs/DesignDocs/AttestationQPLEnforcementSupport.md

```diff
@@ -0,0 +1,86 @@
+Attestation QPL Enforcement Support
+====
+
+This proposal is to make a change to `oe_verify_evidence`, so that it can support policy based QPL (Quote Provider Library) enhancement, such as allowing the API user to pass data to QPL for more advanced quote verification.
```

Comment:
```suggestion
This proposal is to make a change to `oe_verify_evidence`, so that it can support policy based SGX QPL (Quote Provider Library) enhancement, such as allowing the API user to pass data to the SGX QPL for more advanced quote verification.
```

Reply:
s/policy based/policy-based